src/test/scala/io/altoo/akka/serialization/kryo/CryptoSerializationTest.scala

```diff
@@ -11,40 +12,49 @@ import scala.collection.immutable.HashMap
 object CryptoSerializationTest {
   private val config =
     """
-    akka {
-      actor {
-        serializers {
-          kryo = "io.altoo.akka.serialization.kryo.KryoSerializer"
-        }
-        serialization-bindings {
-          "scala.collection.immutable.HashMap" = kryo
-          "[Lscala.collection.immutable.HashMap;" = kryo
-          "scala.collection.mutable.LongMap" = kryo
-          "[Lscala.collection.mutable.LongMap;" = kryo
-        }
-      }
-    }
-    akka-kryo-serialization {
-      post-serialization-transformations = aes
-      encryption {
-        aes {
-          key-provider = "io.altoo.akka.serialization.kryo.DefaultKeyProvider"
-          mode = "AES/GCM/PKCS5Padding"
-          iv-length = 12
-          password = "j68KkRjq21ykRGAQ"
-          salt = "pepper"
-        }
-      }
-    }
-    """
+      |akka {
+      |  actor {
+      |    serializers {
+      |      kryo = "io.altoo.akka.serialization.kryo.KryoSerializer"
+      |    }
+      |    serialization-bindings {
+      |      "scala.collection.immutable.HashMap" = kryo
+      |      "[Lscala.collection.immutable.HashMap;" = kryo
+      |      "scala.collection.mutable.LongMap" = kryo
+      |      "[Lscala.collection.mutable.LongMap;" = kryo
+      |    }
+      |  }
+      |}
+      |akka-kryo-serialization {
+      |  post-serialization-transformations = aes
+      |  encryption {
+      |    aes {
+      |      key-provider = "io.altoo.akka.serialization.kryo.DefaultKeyProvider"
+      |      mode = "AES/GCM/PKCS5Padding"
+      |      iv-length = 12
+      |      password = "j68KkRjq21ykRGAQ"
+      |      salt = "pepper"
+      |    }
+      |  }
+      |}
+      |""".stripMargin
```

Comment:
| and stripMargin just makes this harder to read... are there any reasons to change it here?